Doc/library/importlib.rst

```diff
@@ -487,13 +492,20 @@ ABC hierarchy::
     expected to be a :term:`path-like object` which represents
     conceptually just a file name. This means that no subdirectory
     paths should be included in the *resource* argument. This is
-    because the location of the package that the loader is for acts
-    as the "directory". Hence the metaphor for directories and file
+    because the location of the package the reader is for acts as the
+    "directory". Hence the metaphor for directories and file
     names is packages and resources, respectively. This is also why
     instances of this class are expected to directly correlate to
     a specific package (instead of potentially representing multiple
     packages or a module).
 
+    Loaders that wish to support resource reading are expected to
```

Comment:
What about:

"... which returns an object implementing this ABC's interface.  If the module specified by *fullname* is not a package, this method should return :const:None"  ...?